UI-Assignment-8/Queue.js

```diff
@@ -0,0 +1,28 @@
+function Queue() {
+    this._oldestIndex = 1;

```

Comment:
Use front and rear to store the indices.
Why aren't we starting from 0 index position?